### ISSUE TITLE
JSX style is unreadable and too dark

### ISSUE THREAD

**Issue by dbkaplun** [simurai/duotone-syntax#22]
![screen shot 2016-11-19 at 1 04 52 pm](https://cloud.githubusercontent.com/assets/1765130/20458405/585c7efa-ae59-11e6-8b49-930b294068e5.png)
[Code in question](https://github.com/dbkaplun/hough.js/blob/master/src/index.jsx#L48-L72)

**Comment by simurai** [comment]
@dbkaplun Thanks for example code and screenshot.. that's very helpful.

Just another question: What's the language package you are using? I can see the grammar as `JavaScript (JSX).

**Comment by dbkaplun** [comment]
I'm using `linter-jsxhint@0.1.2`, is that what you mean? That's the only package I see when I search for "jsx" in the Packages tab of Settings. Maybe it's built-in?

**Comment by simurai** [comment]
`linter-jsxhint@0.1.2` shouldn't do syntax highlighting. Do you have https://atom.io/packages/language-babel. It has `jsx` support and is quite popular.

**Comment by dbkaplun** [comment]
I have package `react@0.16.1`, maybe that's it? It's way too dark with .js as well. I'll attach a screenshot when I get the chance.

**Comment by simurai** [comment]
Yes, `react` is probably it. I'll try it out with your sample code and see why it's so dark.

**Comment by dbkaplun** [comment]
Here's some nice [dark JS code](https://github.com/sourcelair/xterm.js/blob/09d67a6c52b0f2018820af34a87fcb8f24fc9d6b/src/xterm.js#L175-L184)
![screen shot 2016-11-25 at 5 45 42 pm](https://cloud.githubusercontent.com/assets/1765130/20637268/23c9b51c-b337-11e6-8153-462f2c1e182d.png)

**Comment by simurai** [comment]
Ok, this should be improved with https://github.com/simurai/duotone-syntax/pull/21 and published as `2.0.1`.

![screen shot 2016-11-26 at 10 25 56 pm](https://cloud.githubusercontent.com/assets/378023/20640660/544a5872-b427-11e6-8f91-30a999ac7685.png)

**Comment by dbkaplun** [comment]
Definitely looks a little better, still very dark though (specifically punctuation and builtins like `moveTo` and `Math.sin`). Even `linesCtx` is very dark. It's a shame because this is my favorite theme and I hate having to use other themes. Should I open another issue? Looks like your Atom is significantly redder than mine.. on your screen, `beginPath` is white and on mine it's lavender. Maybe that has something to do with it?
![screen shot 2016-11-26 at 9 20 32 am](https://cloud.githubusercontent.com/assets/1765130/20642029/bb8001cc-b3b9-11e6-8aa4-925931fc91b9.png)

**Comment by simurai** [comment]
> Looks like your Atom is significantly redder than mine.. on your screen, beginPath is white and on mine it's lavender. Maybe that has something to do with it?

In DuoTone dark you can now change the colors in the settings, see: https://github.com/simurai/duotone-dark-syntax#settings So you could brighten everything.

I'll see about brightening the default since that seems a bit dark.
